.github/workflows/stale.yaml

```diff
@@ -0,0 +1,16 @@
+name: 'Close stale issues and PRs'
+on:
+  schedule:
+    - cron: '0 0 * * *'
+
+jobs:
+  stale:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/stale@v8
+        with:
+          stale-issue-label: 'stale'
+          stale-issue-message: '  This issue has been automatically marked as stale because it has not had recent activity. It will be closed if no further activity occurs. Thank you for your contributions.'
```

Comment:
Why the two spaces at the start of the message?

Reply:
I have removed the spaces.